corehq/apps/sms/templates/sms/usage_report.html

```diff
@@ -0,0 +1,15 @@
+{% extends 'hqwebapp/base_section.html' %}
+{% load i18n %}
+{% load hq_shared_tags %}
+{% load timezone_tags %}
+{% requirejs_main "sms/js/sms_usage_report" %}
+{% block page_content %}
+    <form method="post">
+        {% csrf_token %}
+        <div class="form-group">
+            <label for="date_range_selector">Select Date Range</label>
+            <input name="date_range" class="form-control" id="date_range_selector" data-startDate="" data-endDate="" placeholder="Date">
+        </div>
+        <button class="btn btn-default">Generate Report</button>
```

Comment:
`btn-primary` instead of `btn-default`, because this is the major action you want the user to take.

Reply:
Done.